pkg/cmd/server/kubernetes/node_config.go

```diff
@@ -280,3 +288,30 @@ func BuildKubernetesNodeConfig(options configapi.NodeConfig) (*NodeConfig, error
 
 	return config, nil
 }
+
+// ensureDockerAPIVersionForParallelImagePulls returns true if the docker API version
+// is >=1.21, else false.
+func ensureDockerAPIVersionForParallelImagePulls(dh *docker.Helper, version string) bool {
```

Comment:
I am trying to understand, BuildKubernetesNodeConfig calls this ensureDockerAPIVersionForParallelImagePulls, which takes helper and create the client. If BuildKubernetesNodeConfig creates the client by the helper, and the pass the client to  ensureDockerAPIVersionForParallelImagePulls, what would be the net benefit?